test/acceptance/spec/22-marriage-details.js

```diff
@@ -1,7 +1,9 @@
 'use strict';
 
-var expectedRecord = require('../expected-marriage-record');
-var expectedRecords = require('../expected-marriage-records');
+const expectedRecord = require('../expected-marriage-record');
+const expectedRecords = require('../expected-marriage-records');
+const role = require('rewire')('../../../controllers/marriage-details') // eslint-disable-line no-underscore-dangle
```

Comment:
I think it would be better to either hardcode this or move it out to `config.js` rather than use rewire.

Reply:
Agreed - I'll include this change as well.